js/app/src/lite/LiteIndex.svelte

```diff
@@ -181,7 +182,7 @@
 			<Index
 				space={null}
 				src={null}
-				host={null}
+				host={FAKE_LITE_HOST}
```

Comment:
This `host` is passed to the client object through the code below
https://github.com/gradio-app/gradio/blob/9b42ba8f1006c05d60a62450d3036ce0d6784f86/js/app/src/Index.svelte#L276-L287

and the client makes an HTTP call if the URL matches HF Spaces' host here, which causes the error:
https://github.com/gradio-app/gradio/blob/9b42ba8f1006c05d60a62450d3036ce0d6784f86/client/js/src/client.ts#L235-L238

So we explicitly set the different value to `host`, which is also considered as a special host name representing the Lite's server.

Reply:
@pngwn Should we use `src` rather than `host`?  #3065

Reply:
That is a breaking change, so it will need to happen in 5.0.

Reply:
@pngwn Thanks but do you mean we need to wait for this to be merged until 5.0, or just using `host` not `src` is ok and we can merge this?

Reply:
Should be merged after https://github.com/gradio-app/gradio/pull/9102?